app/styles/main.scss

```diff
@@ -23,6 +23,10 @@ html, body {
   height: 100%;
 }
 
+.body-container {
+  min-height: calc(100vh - 317px);
```

Comment:
My real fear is just maintainability vs value. A sticky footer is a nice to have, but maintainable code is a must-have.

Anyway, that's why I'm glad to have seen the PR so we can see what the tradeoff looks like. I think the tradeoff is too much. Would you agree?